11-payment-encoding.md

```diff
@@ -64,6 +64,7 @@ A writer:
   - if a specific minimum `amount` is required for successful payment:
 	  - MUST include that `amount`.
 	- MUST encode `amount` as a positive decimal integer with no leading 0s.
+	- If the `p` multiplier is used the `amount` the last decimal MUST be `0`.
```

Comment:
nit: @cdecker I think you missed a last comment that this sentence is weird.
It should be: "If the `p` multiplier is used the last decimal of the `amount` MUST be `0`."

Reply:
Oh damn, I thought there was a fixup from @Sword-Smith that fixed that. I'll open a new PR with the spelling mistake.

Reply:
I think I did leave a fixup. But for some reason it didn't get included. I made a new PR
https://github.com/lightningnetwork/lightning-rfc/pull/733